packages/cosmic-swingset/Makefile

```diff
@@ -145,7 +145,7 @@ t1/decentral-economy-config.json: $(ECONOMY_PROPOSALS)
 		'$(ECONOMY_PROPOSALS)' > t1/decentral-economy-config.json
 
 scenario2-run-chain-economy: t1/decentral-economy-config.json
-	echo "DEPRECATED use: CHAIN_BOOTSTRAP_VAT_CONFIG=@agoric/vats/decentral-test-vaults-config.json scenario2-run-chain"
+	echo "DEPRECATED use: CHAIN_BOOTSTRAP_VAT_CONFIG=@agoric/vats/decentral-unittest-vaults-config.json scenario2-run-chain"
```

Comment:
this config is only a unit when testing the config per se. any use of it to instantiate vats is an integration test.

I get the `decentral-test-vaults-config` name is taken so you need another. here are some ideas:
- `decentral-btest-vaults-config`
- `testing-vaults-config`
- `decentral-test-config`
- `test-config`

Reply:
Yeah, I tend to think of its purpose as integration testing. But to avoid too strong a binding, I'm mixing it with the `btest` suggestion to get `itest`.

Reply:
itest, utest, we all test!